taskcluster/ci/signing/kind.yml

```diff
@@ -49,13 +49,14 @@ job-template:
             # No test job runs on push against this build type. Although we do want nightly-simulation
             # signed to use it in the gecko trees.
             # We want beta on push so that we detect problem before shipping it
-            (nightly-simulation|beta-firebase|android-test-beta): [github-push]
+            (nightly-simulation|beta-firebase|android-test-beta|nightly-firebase|android-test-nightly): [github-push]
```

Comment:
@bhearsum I forgot to add this in previous PR that failed, do you think this would make any difference to avoid the issue I had before? 

This was the previous [PR](https://github.com/mozilla-mobile/fenix/pull/17513) and the [issue](https://firefox-ci-tc.services.mozilla.com/tasks/babVJdOlSmK3DlsGUlCbSw/runs/1/logs/https%3A%2F%2Ffirefox-ci-tc.services.mozilla.com%2Fapi%2Fqueue%2Fv1%2Ftask%2FbabVJdOlSmK3DlsGUlCbSw%2Fruns%2F1%2Fartifacts%2Fpublic%252Flogs%252Flive.log%3Fbewit%3DbW96aWxsYS1hdXRoMC9hZHxNb3ppbGxhLUxEQVB8aXJpb3NcMTYxMjk2NTg3M1x1NnBsaWFGR1RUdHYwNlJyM0tsclUzcnlOOWpHeU5neWhzdHIzQnlmVGxVPVxleUpqWlhKMGFXWnBZMkYwWlNJNmV5SjJaWEp6YVc5dUlqb3hMQ0p6WTI5d1pYTWlPbHNpWVhOemRXMWxPbTF2ZW1sc2JHRXRaM0p2ZFhBNlNXNTBjbUZ1WlhSWGFXdHBJaXdpWVhOemRXMWxPbTF2ZW1sc2JHRXRaM0p2ZFhBNlUzUmhkSE5FWVhOb1ltOWhjbVFpTENKaGMzTjFiV1U2Ylc5NmFXeHNZUzFuY205MWNEcGhiR3hmYkdSaGNGOTFjMlZ5Y3lJc0ltRnpjM1Z0WlRwdGIzcHBiR3hoTFdkeWIzVndPbU5zYjNWa2MyVnlkbWxqWlhOZlpHVjJJaXdpWVhOemRXMWxPbTF2ZW1sc2JHRXRaM0p2ZFhBNlpXZGxibU5wWVY5MWF5SXNJbUZ6YzNWdFpUcHRiM3BwYkd4aExXZHliM1Z3T20xdlltbHNaVjl5Wld4bFlYTmxjeUlzSW1GemMzVnRaVHB0YjNwcGJHeGhMV2R5YjNWd09tOW1abWxqWlY5eVpXMXZkR1VpTENKaGMzTjFiV1U2Ylc5NmFXeHNZUzFuY205MWNEcHdhRzl1WldKdmIydGZZV05qWlhOeklpd2lZWE56ZFcxbE9tMXZlbWxzYkdFdFozSnZkWEE2YzJWeWRtbGpaVjlxYVhKaElpd2lZWE56ZFcxbE9tMXZlbWxzYkdFdFozSnZkWEE2ZEdWaGJWOXRiMk52SWl3aVlYTnpkVzFsT20xdmVtbHNiR0V0WjNKdmRYQTZkbkJ1WDJOdmNuQWlMQ0poYzNOMWJXVTZiVzk2YVd4c1lTMW5jbTkxY0RwMmNHNWZaR1ZtWVhWc2RDSXNJbUZ6YzNWdFpUcHNiMmRwYmkxcFpHVnVkR2wwZVRwdGIzcHBiR3hoTFdGMWRHZ3dMMkZrZkUxdmVtbHNiR0V0VEVSQlVIeHBjbWx2Y3lKZExDSnpkR0Z5ZENJNk1UWXhNamsyTkRBM01qZ3hOU3dpWlhod2FYSjVJam94TmpFeU9UWTFPRGN5T0RFMUxDSnpaV1ZrSWpvaVUzZHBjMVV6YWtaVUxVOWlTbXQ2ZGxsNFZISlNkelZUU0hkM1VGbHdVWFZYYTI1dFIydzFaazl5ZW5jaUxDSnphV2R1WVhSMWNtVWlPaUpzTlVobFFrVllXaXRUVUZVcldEZGFSSEEzUW5rMmVURnJiblI2VTA5WU5ISnRSRUV4UkdOdGMxcFpQU0lzSW1semMzVmxjaUk2SW5OMFlYUnBZeTkwWVhOclkyeDFjM1JsY2k5M1pXSXRjMlZ5ZG1WeUluMTk)

Reply:
I think this may fix the issue; it's worth a try.

Reply:
Let's land it and try...if it fails I will revert it right away